<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="PCW10MAOC" failures="2" tests="2" name="Default test(failed)" time="7.309" errors="0" timestamp="11 jun 2018 23:39:38 GMT+00:00">
  <testcase classname="automationFramework.EjecutaPrueba" name="Login" time="7.293">
    <failure type="java.lang.NoClassDefFoundError" message="org/junit/Assert">
      <![CDATA[java.lang.NoClassDefFoundError: org/junit/Assert
at automationFramework.EjecutaPrueba.Login(EjecutaPrueba.java:73)
Caused by: java.lang.ClassNotFoundException: org.junit.Assert
at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(Unknown Source)
at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(Unknown Source)
at java.base/java.lang.ClassLoader.loadClass(Unknown Source)
... 25 more
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- Login -->
  <testcase classname="automationFramework.EjecutaPrueba" name="case2" time="0.002">
    <failure type="java.lang.AssertionError" message="expected [12] but found [11]">
      <![CDATA[java.lang.AssertionError: expected [12] but found [11]
at automationFramework.EjecutaPrueba.case2(EjecutaPrueba.java:108)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- case2 -->
</testsuite> <!-- Default test(failed) -->
